notebooks/Emukit-task-spread-of-disease-with-seir-model.ipynb

```diff
@@ -0,0 +1,908 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Spread of an infectious disease: SEIR-model with Emukit\n",
+    "\n",
+    "This notebook uses the SEIR-model and Emukit to answer several application driven research question."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# General imports\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "from matplotlib import colors as mcolors\n",
+    "\n",
+    "# Figure config\n",
+    "colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)\n",
+    "LEGEND_SIZE = 15\n",
+    "TITLE_SIZE = 25\n",
+    "AXIS_SIZE = 15\n",
+    "FIGURE_SIZE = (12, 8)\n",
+    "\n",
+    "# for reproducability\n",
+    "np.random.seed(42)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## The SEIR epidemic model\n",
+    "\n",
+    "The [SEIR model](https://en.wikipedia.org/wiki/Compartmental_models_in_epidemiology#The_SEIR_model) is a compartmental model of biology that simulates the spread of an infectious disease. \n",
```

Comment:
I think it is worth spending some more time here, explaining what this model is about. The way it stands right now it seems to assume some knowledge about at least SIR model in general.

Something like "everyone is assumed suspectible. as soon as people contract an infection they transition to E and then to I. once diseases is over they transition to R and become immune to this infection. model is parameterized by a and b etc. etc.". This would help people completely unfamiliar with the topic understand mechanics of the model.

Reply:
Good point. I'll add some sentences.